lib/system.nim

```diff
@@ -2339,8 +2339,7 @@ when notJSnotNims:
   when hostOS != "standalone" and hostOS != "any":
     include "system/dyncalls"
 
-  import system/countbits_impl
-  include "system/sets"
+  from system/sets import cardSet
```

Comment:
Undo this change please, `system/sets` is now only an include file

Reply:
OK, will finish it later